Rlabkey/R/labkey.webdav.R

```diff
@@ -0,0 +1,230 @@
+##
+#  Copyright (c) 2019 LabKey Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+##
+
+labkey.webdav.get <- function(baseUrl=NULL, folderPath, fileSet="@files", remoteFilePath, localFilePath, overwrite=TRUE)
+{
```

Comment:
if you are using default values, they are often more useful if they are at the end of the parameter list, especially if the user is matching parameters by position.

Reply:
sure, good point.  just added that.